src/common/virtualized/tableRow.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+
+// https://github.com/bvaughn/react-virtualized/blob/master/docs/Table.md
+// https://github.com/bvaughn/react-virtualized/blob/master/source/Table/defaultRowRenderer.js
+export interface TableRowProps {
+  className: string;
+  style: React.CSSProperties;
+  columns: any[];
+  index: number;
+  key: any;
+  isScrolling: boolean;
+  onRowClick?: () => void;
+  onRowDoubleClick?: () => void;
+  onRowMouseOver?: () => void;
+  onRowMouseOut?: () => void;
+}
+
+export const TableRowComponent: React.StatelessComponent<TableRowProps> = (props) => {
+  return (
+    <div className={props.className} style={props.style}>
+      {React.Children.toArray(props.children).map(cellRenderer(props))}
+    </div>
+  );
+};
+
+const cellRenderer = ({columns}) => (cell, index) => {
```

Comment:
Proposal: rename to `TableCellComponent` or `CellComponent`?

Does it make sense to move to other file?